docker/start-bootnode.sh

```
@@ -24,30 +36,43 @@ for env in PRIVATE_KEY; do
 
-echo "🥾 Starting the bootnode with address at $address"
-echo $address > ./bootnodes/bootnodes.list
+if [[ ! -z $VERBOSITY ]]; then
+  params="$params -verbosity ${VERBOSITY}"
+fi
 
+host=$(hostname -i | awk '{print $1}')
+if [[ -z "${DISABLE_EXTIP:-}" && -n "${EXTIP:-}" ]]; then
+  host=$EXTIP
+fi
+address="enode://$(bootnode -nodekey ${NODEKEY_FILE} -writeaddress)@${host}:${BOOTNODE_PORT}"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)